redbeams/src/main/java/com/sequenceiq/redbeams/RedbeamsApplication.java

```diff
@@ -9,7 +9,7 @@
 @EnableSwagger2
 @SpringBootApplication(scanBasePackages = {"com.sequenceiq.redbeams",
         "com.sequenceiq.cloudbreak.auth.altus",
-        "com.sequenceiq.cloudbreak.auth.security",
```

Comment:
I think com.sequenceiq.cloudbreak.auth.security needs to remain because that brings in ScimAccountGroupReaderFilter (see SecurityConfig). Does redbeams still start successfully with this removal?

Reply:
yes, actually this was needed for another required filter